src/main/java/com/redhat/devtools/lsp4ij/LanguageServiceAccessor.java

```diff
@@ -39,6 +39,10 @@
 
 /**
  * Language server accessor.
+ *
+ * This method should be never used because it is an internal class.

```

Comment:
This class is considered internal and should not be used in other plugins.

Reply:
fixed